src/components/TileCatalog/TilePagination/TilePagination.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const propTypes = {
+  page: PropTypes.number.isRequired,
+  numPages: PropTypes.number.isRequired,
+  onChange: PropTypes.number.isRequired,
+  i18n: PropTypes.shape({}),
+};
+const defaultProps = {
+  i18n: {
+    ariaLabelPreviousPage: 'Previous page',
+    ariaLabelNextPage: 'Next page',
+    ariaLabelPagination: 'pagination',
+    ariaLabelPage: 'page',
+  },
+};
+
+const TilePagination = ({ page, numPages, onChange, i18n }) => {
+  const prevButton = (
```

Comment:
Since there's not much logic in these buttons, it's easier to read if you put them all into the return below, but thats just a personal preference

Reply:
I actually disagree in this particular case, since we're using vanilla JS from the carbon simple pagination (which doesn't have a react wrapper yet) it's a bit verbose to keep all in the render method.  I prefer keeping the render method pretty sparse so that you can more easily see the layout.  (but it's also a personal preference 👍)

Reply:
Good point. Code on!